### ISSUE TITLE
Failed to cross compile resnet50_v1 for tvm.target.mali ARMv7 Mali GPU

### ISSUE THREAD

**Issue by apivovarov** [dmlc/tvm#1820]
I tried to cross compile `resnet50_v1` model for `tvm.target.mali()` ARMv7 Mali GPU (I run the compilation on x86_64 Ubuntu 16.04 box)

The code
https://gist.github.com/apivovarov/db3275344b013678fc1cb9e187aecfae

Error:
```
model_name:  resnet50_v1 , target: opencl -device=mali -model=unknown , target_host:  llvm -target=armv7l-linux-gnueabihf , opt_level: 3 , data_shape: (1, 3, 224, 224)
Compiling...
Traceback (most recent call last):
  File "./compile.py", line 40, in <module>
    graph, lib, params = nnvm.compiler.build(sym, target=target, shape={"data": data_shape}, params=params, target_host=target_host)
  File "/usr/local/lib/python3.5/dist-packages/nnvm-0.8.0-py3.5.egg/nnvm/compiler/build_module.py", line 281, in build
    graph = optimize(graph, shape, dtype, layout)
  File "/usr/local/lib/python3.5/dist-packages/nnvm-0.8.0-py3.5.egg/nnvm/compiler/build_module.py", line 170, in optimize
    graph = graph.apply(["InferShape", "InferType", "AlterOpLayout"])
  File "/usr/local/lib/python3.5/dist-packages/nnvm-0.8.0-py3.5.egg/nnvm/graph.py", line 234, in apply
    check_call(_LIB.NNGraphApplyPasses(self.handle, npass, cpass, ctypes.byref(ghandle)))
  File "/usr/local/lib/python3.5/dist-packages/nnvm-0.8.0-py3.5.egg/nnvm/_base.py", line 75, in check_call
    raise NNVMError(py_str(_LIB.NNGetLastError()))
nnvm._base.NNVMError: TVMCall CFunc Error:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/tvm-0.5.dev0-py3.5-linux-x86_64.egg/tvm/_ffi/_ctypes/function.py", line 55, in cfun
    rv = local_pyfunc(*pyargs)
  File "/usr/local/lib/python3.5/dist-packages/nnvm-0.8.0-py3.5.egg/nnvm/top/nn.py", line 150, in alter_conv2d_layout
    return topi.nn.conv2d_alter_layout(attrs, inputs, tinfos)
  File "<decorator-gen-26>", line 2, in conv2d_alter_layout
  File "/usr/local/lib/python3.5/dist-packages/tvm-0.5.dev0-py3.5-linux-x86_64.egg/tvm/target.py", line 356, in dispatch_func
    return dispatch_dict[k](*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/topi-0.5.dev0-py3.5.egg/topi/arm_cpu/conv2d.py", line 537, in _alter_conv2d_layout
    new_attrs['kernel_layout'] = 'OIHW%do' % (cfg['tile_co'].size[-1])
  File "/usr/local/lib/python3.5/dist-packages/tvm-0.5.dev0-py3.5-linux-x86_64.egg/tvm/autotvm/task/space.py", line 758, in __getitem__
    return self._entity_map[name]
KeyError: 'tile_co'
```

**Comment by tqchen** [comment]
Thanks for raising the issue, for general usage questions, please open a thread at https://discuss.tvm.ai/

**Comment by apivovarov** [comment]
@tqchen I think it is not "general usage" question. I found quite specific error with resnet50 models compilation for ARM+Mali devices. I think it is an issue.

**Comment by tqchen** [comment]
Well, I agree that there is a blurred line here. The community will be working together to solve the problem both in the issue tracker and the forum:) sometimes forum is even better than issue as the active thread get promoted to the top. We try to keep the issues minimum and actionable so all of them get attended to

**Comment by merrymercy** [comment]
It should already be fixed by #1796. Is your version before that fix?
